src/software/ai/passing/pass_generator_test.cpp

```diff
@@ -194,12 +188,11 @@ TEST_F(PassGeneratorTest, test_passer_point_changes_are_respected)
     stepPassGenerator(pass_generator, *world, 100);
 
     // Find what pass we converged to
-    auto pass_evaluation = pass_generator->generatePassEvaluation(*world);
-    auto converged_pass  = pass_evaluation.getBestPassOnField().pass;
+    auto converged_pass = pass_generator.getBestPass(*world).pass;
 
-    // We expect to have converged to a point closer to the robot in the neg_y
-    // compared to the robot in the pos_y position.
-    EXPECT_GT((converged_pass.receiverPoint() - pos_y_friendly.position()).length(),
+    // We expect to have converged to a point closer to the robot in the pos_y
```

Comment:
is this comment consistent with the assertion?

Reply:
I think so, the ball is in +y, so the expectation is that the pass receiver should also be close to the robot in +y, as opposed to the one in -y.

Reply:
What I'm having trouble understanding is that why does it make sense to pass to the robot that's closest to us? This seems like what this test is trying to do.